### PR TITLE
Simple Payments: add a Guided Tour for end of the year

### PR DESCRIPTION
**Pull request by lsinger**
This PR adds a new tour, telling people about the Simple Payment Button. It should start for people on the Business or Premium plan when they're visiting a page on which the sidebar's "Site Pages" menu item (and it's associated "Add" button) is visible. 

Note: the centered step that comes up when we enter the Editor is there for technical reasons. Unfortunately the Editor takes too long to load / Guided Tours is too impatient to find the target element to attach the step to. 

To launch the tour:

http://calypso.localhost:3000/stats/day/SITE_URL?tour=simplePaymentsEndOfYearGuide

Screen capture:

happy-holidays

To test:
- launch the tour
- check the copy and flow of the tour
- try to break the tour
- make sure the code looks alright

### CODE REVIEW

File: client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+						},
+					}
+				) }
+			</p>
+			<img
+				src="https://i0.wp.com/en-support.files.wordpress.com/2017/03/editor-content-area_360.gif"
+				style={ { marginBottom: '10px', border: '3px solid #00AADC', borderRadius: '4px' } }
+			/>
+			<ButtonRow>
+				<Next step="add-new-page">{ translate( 'Get started!' ) }</Next>
```
Comment (a8ci18n) [inline client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js]: Hi! I've found a possible matching string that has already been translated **89** times:
`translate( 'Get Started' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/guided-tour-end-of-year)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kristastevens** [comment]
Hi @lsinger thanks again for reaching out. Here are some copy suggestions: 

Tip 1:
"Prepare for holiday shopping and end-of-year donations: add a payment button to sell products and services or accept donations  on your site!"

* I removed reference to Cyber Monday as I don't think it's ubiquitously known -- even across the business section of customers we're targeting here.
* On this note -- I don't think we need the "quick brown fox" animation. Can we remove it? 

Tip 2: 
"To add a payment button, create a page. Click <strong>Add</strong> to continue." 

Tip 3: 
"Add a payment button here, from within the Editor." 

* Slight clarification here, just in case the user doesn't know that they are actually in the Editor at this moment. 

Tip 4: 
Looks great! No suggested changes. 

Tip 5: 
Looks great! No suggested changes. 

Tip 6: 
Line one: looks great! No suggested changes. 

Line two:
"Remember: add your page to your site navigation so visitors can find it easily. Learn about <a href="https://en.support.wordpress.com/menus/">managing menus</a>. 

* Line two -- we try to reinforce with a positive, "remember, vs. don't forget." Added a link to help for menus, just in case.

**Comment by iamtakashi** [comment]
Thanks for working on this, @lsinger !

I was wondering if it's better or possible this step moves automatically to the next when you add a payment button. No worries if it complicates and delays the shipping though.

screen shot 2017-11-03 at 14 37 56

**Comment by marekhrabe** [comment]
The flow looks good to me, with the exception for the same thing as Takashi mentioned. The modal window is self-explanatory so I wasn't expecting step there but after closing it, I'd expect the step that points to "Add New" to disappear. However, it's a nice to have change, definitely not a blocker. 

This is a tour that I would really love to be linkable from support docs and from HEs :) Great job on planning & dev! 🎉

**Comment by iamtakashi** [comment]
The copy has been updated as suggested. Thank you, @kristastevens !

**Comment by kristastevens** [comment]
Happy to be of service! Thanks for reaching out!

File: client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+				) }
+			</p>
+			<div style={ { textAlign: 'center' } }>
+				<img
+					src="/calypso/images/illustrations/illustration-shopping-bags.svg"
+					style={ { width: '210px', marginBottom: '10px' } }
+				/>
+			</div>
+			<ButtonRow>
+				<Next step="add-new-page">{ translate( 'Get started!' ) }</Next>
```
Comment (a8ci18n) [inline client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js]: Hi! I've found a possible matching string that has already been translated **89** times:
`translate( 'Get Started' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/guided-tour-end-of-year)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by lsinger** [comment]
Copy and illustration have been updated. Good to go from our end.

File: client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+				<Quit>{ translate( 'No thanks.' ) }</Quit>
+			</ButtonRow>
+			<Link href="https://learn.wordpress.com/get-published/">
```
[the remark this answers is not in the document]
Comment (lsinger) [reply client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js]: Addressed in 15359a3.

File: client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<div style={ { textAlign: 'center' } }>
+				<img
+					src="/calypso/images/illustrations/illustration-shopping-bags.svg"
+					style={ { width: '210px', marginBottom: '10px' } }
```
[the remark this answers is not in the document]
Comment (lsinger) [reply client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js]: Addressed in 15359a3.

**Comment by marekhrabe** [comment]
I cleared my seen tours and started Calypso in Reader. Once I navigated to my Business site stats, this tour correctly showed up.

screen shot 2017-11-04 at 17 19 36

File: client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+				) }
+			</p>
+			<div style={ { textAlign: 'center' } }>
+				<img
+					src="/calypso/images/illustrations/illustration-shopping-bags.svg"
+					style={ { width: '210px', height: '160px', marginBottom: '10px' } }
+				/>
+			</div>
+			<ButtonRow>
+				<Next step="add-new-page">{ translate( 'Get started!' ) }</Next>
```
Comment (a8ci18n) [inline client/layout/guided-tours/tours/simple-payments-end-of-year-guide.js]: Hi! I've found a possible matching string that has already been translated **89** times:
`translate( 'Get Started' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/guided-tour-end-of-year)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by dechov** [comment]
My experience when seeing "Add a payment button here, from within the Editor." — after the very specific instruction in the previous step, I thought it was instructing me to add a payment button before clicking Next. Should it say "click Next to see how" or something similar, so the user is assured the tour will proceed to show them exactly where in the UI the button can be added?

**Comment by iamtakashi** [comment]
> I thought it was instructing me to add a payment button before clicking Next.

@dechov Thanks for sharing your experience! There is no significant drop out on the step so far — 79.8% of people who saw the step went to the next step. However, I can understand the copy can be perceived in that way. 

How about "Welcome to the Editor, click _Next_ to see how to add a payment button."?

cc @kristastevens  @lsinger

**Comment by kristastevens** [comment]
Hi @iamtakashi, that sounds good with a couple small changes: 

"Welcome to the Editor! Click _Next_ to learn how to add a payment button."

**Comment by iamtakashi** [comment]
Perfect. Thanks, @kristastevens!

**Pull request merged**